qiita_db/data.py

```diff
@@ -1144,6 +1144,21 @@ def study(self):
             "processed_data_id=%s".format(self._study_processed_table),
             [self._id])[0]
 
+    @property
+    def samples(self):
```

Comment:
Ah, this is a vestige of the old way of finding processed data samples. Do we want to keep this function still?